src/mission_companion.cpp

```diff
@@ -185,28 +185,28 @@ bool talk_function::outpost_missions(npc *p, std::string id, std::string title)
     std::string entry, entry_aux;
 
     if (id == "SCAVENGER"){
-        col_missions["Assign Scavenging Patrol"] = "Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
+        col_missions["Assign Scavenging Patrol"] = _("Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
```

Comment:
A couple questions to anyone who's possible to answer on them:
1) is it a workable method to translate this bunch of text lines?
2) do we have any easy way to test missions?